commands/local_server_start.go

```diff
@@ -287,11 +287,16 @@ var localServerStartCmd = &console.Command{
 			scheme = "http"
 		}
 
+		address := config.ListenIp
+		if c.Bool("allow-all-ip") {
```

Comment:
in a close future we might have other ways than this flag to define this behavior so I would rather check the address directly:

```suggestion
		if address == “” {
```